packages/region/src/services/region-module.ts

```diff
@@ -325,3 +415,35 @@ export default class RegionModuleService<
     }
   }
 }
+
+// microORM complains if undefined fields are present in the passed data object
+const removeUndefined = <T extends Record<string, any>>(obj: T): T => {
+  return Object.fromEntries(
+    Object.entries(obj).filter(([_, v]) => v !== undefined)
+  ) as T
+}
+
+const asMap = <T extends Record<string, any>>(
```

Comment:
suggestion: rm not sure it is necessary as creating a Map object is equivalent and one liner